addons/arsenal/functions/fnc_updateUniqueItemsList.sqf

```diff
@@ -260,16 +264,21 @@ private _itemInfoType = 0;
                 };
                 // Unknown
                 default {
-                    // Don't add items that are part of the arsenal
-                    private _attachments = GVAR(virtualItems) get IDX_VIRT_ATTACHMENTS;
+                    // Don't add attachments or misc. items
                     if (
-                        !(_x in (_attachments get IDX_VIRT_OPTICS_ATTACHMENTS)) &&
-                        {!(_x in (_attachments get IDX_VIRT_FLASHLIGHT_ATTACHMENTS))} &&
-                        {!(_x in (_attachments get IDX_VIRT_MUZZLE_ATTACHMENTS))} &&
-                        {!(_x in (_attachments get IDX_VIRT_BIPOD_ATTACHMENTS))} &&
+                        !(_baseWeapon in (_attachments get IDX_VIRT_OPTICS_ATTACHMENTS)) &&
+                        {!(_baseWeapon in (_attachments get IDX_VIRT_FLASHLIGHT_ATTACHMENTS))} &&
+                        {!(_baseWeapon in (_attachments get IDX_VIRT_MUZZLE_ATTACHMENTS))} &&
+                        {!(_baseWeapon in (_attachments get IDX_VIRT_BIPOD_ATTACHMENTS))} &&
                         {!(_x in (GVAR(virtualItems) get IDX_VIRT_MISC_ITEMS))}
                     ) then {
-                        (GVAR(virtualItems) get IDX_VIRT_UNIQUE_UNKNOWN_ITEMS) set [_x, nil];
+                        // If item is a weapon (including binos), make it unique
+                        (GVAR(virtualItems) get IDX_VIRT_UNIQUE_UNKNOWN_ITEMS) set [_x,
```

Comment:
I'm not sure if we should update it to `_baseWeapon`, need to think about it further.

Reply:
`_baseWeapon` would break things, sticking with `_x`.